content/fr/docs/concepts/cluster-administration/logging.md

```diff
@@ -0,0 +1,358 @@
+Mon Jan  1 00:00:01 UTC 2001 INFO 1
+Mon Jan  1 00:00:02 UTC 2001 INFO 2
+...
+```
+
+L'agent au niveau du nœud installé dans le cluster récupère les deux flux de
+journaux sans aucune configuration supplémentaire. Il est possible de configurer
+l'agent pour qu'il analyse syntaxiquement les évènements en fonction du
+conteneur source.
+
+Notez que bien que la consommation en CPU et mémoire soit faible ( de l'ordre de
+quelques milicores pour la CPU et quelques mégaoctets pour la mémoire),  ecrire
+les évènements dans un fichier et les envoyer ensuite dans `stdout` peut doubler
+l'espace disque utilisé. Quand une application écrit dans un seul fichier de
+journal il est préférable de configurer `/dev/stdout` comme destination plutôt
+que d'implémenter un conteneur side-car diffusant.
+
+Les conteneurs side-car peuvent être utiliser pour faire la rotation des
+journaux quand l'application n'en est pas capable elle même. Un exemple serait
+un petit conteneur side-car qui effectuerait cette rotation périodiquement.
+Toutefois il est recommandé d'utiliser `stdout` et `stderr` directement et de
+laisser la rotation et les politiques de rétentions au kubelet.
+
+### Conteneur side-car avec un agent de journalisation
+
+![Conteneur side-car avec un agent de
+journalisation](/images/docs/user-guide/logging/logging-with-sidecar-agent.png)
+
+Quand un agent de journalisation au niveau du nœud n'est pas assez flexible pour
+votre utilisation vous pouvez créer un conteneur side-car avec un agent de
+journalisation séparé que vous avez configuré spécialement pour qu'il s'exécute
+avec votre application.
+
+{{< note >}}
+Utiliser un agent de journalisation dans un conteneur side-car peut entraîner
+une consommation de ressource significative. De plus vous n'avez plus accès aux
+journaux avec la commande `kubectl` parce qu'ils ne sont plus gérés par
+kubelet.
+{{< /note >}}
+
+Comme exemple, vous pouvez utiliser
+[Stackdriver](/docs/tasks/debug-application-cluster/logging-stackdriver/) où
+ fluentd est l'agent de journalisation. Ci-dessous se trouve deux
+configurations qui implémentent cette méthode.
+
+Le premier fichier contient un
+[ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) pour
+configurer fluentd.
+
+{{< codenew file="admin/logging/fluentd-sidecar-config.yaml" >}}
+
+{{< note >}}
+La configuration de fluentd est hors du cadre de cet article. Vous trouverez
+des informations pour configurer fluentd dans la [documentation officielle de
+fluentd](http://docs.fluentd.org/).
+{{< /note >}}
+
+Le second fichier est un manifeste pour un Pod avec un conteneur side-car qui
+exécute fluentd. Le Pod monte un volume où fluentd peut récupérer sa
+configuration.
+
+{{< codenew file="admin/logging/two-files-counter-pod-agent-sidecar.yaml" >}}
+
+Apres quelques minutes, les évènements apparaîtront dans l'interface de
+Stackdriver.
+
+Ce n'est qu'un exemple et vous pouvez remplacer fluentd para n'importe quel
```

Comment:
par

Reply:
thanks @feloy for all theses corrections